java/src/jmri/jmrit/withrottle/WiFiConsistManager.java

```diff
@@ -44,9 +45,9 @@ public boolean checkForBroadcastAddress(DccLocoAddress addr) {
     }
 
     @Override
-    public Consist addConsist(DccLocoAddress address) {
+    public Consist addConsist(LocoAddress address) {
         WiFiConsist consist;
-        consist = new WiFiConsist(address);
+        consist = new WiFiConsist((DccLocoAddress) address);
```

Comment:
Before this cast, you need to make sure the address is a DCCLocoAddress.  You should throw an illegal argument exception if it isn't.